docs/source/author/writing_guide.rst

```diff
@@ -253,14 +233,10 @@ See the examples below:
 Link to Heading on Page Within Website
 ------------------------------------------------------------------------------
 
-You can refer to a subsection on a page by following the same syntax as above
-and then adding the subsection name at the end with a ``#`` separator.
-All headers are subsections that have a link that can be linked to (called an
-anchor link).
-The anchor link can be determined by converting the header name to lowercase,
-with spaces replaced with dashes, and punctuation removed.
-In cases where duplicate headings exist on the same page, a number is appended
-on the end of the anchor link.
+You can refer to a subsection on a page by following the same syntax as above and then adding the subsection name at the end with a ``#`` separator.
+All headers are subsections that have a link that can be linked to (called an anchor link).
+The anchor link can be determined by converting the header name to lowercase, with spaces replaced with dashes, and punctuation removed.
+In cases where duplicate headings exist on the same page, a number is appended on the end of the anchor link.
```

Comment:
Is this accurate? The example above is definitely old-style

Reply:
Good catch